src/core/client/stream/classes.ts

```diff
@@ -425,15 +425,7 @@ const CLASSES = {
     /**
      * indentation classes for the different levels.
      */
-    indent: [
-      "coral coral-indent coral-indent-0",
-      "coral coral-indent coral-indent-1",
-      "coral coral-indent coral-indent-2",
-      "coral coral-indent coral-indent-3",
-      "coral coral-indent coral-indent-4",
-      "coral coral-indent coral-indent-5",
-      "coral coral-indent coral-indent-6",
-    ],
+    indent: (level: number) => `coral coral-indent coral-indent-${level}`,
```

Comment:
okay, this is cool, well done you wizard you!

Reply:
hahaha, ty!